src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/comment';
+
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="CommentList">
+    {comments.map(comment => (
+      <li>
```

Comment:
key attribute should be on the parent element here (move it to the li element instead of CommentInfo)